docs/howto/windows.md

```diff
@@ -136,12 +221,24 @@ export PATH="$HOME/bin:$HOME/.local/bin:/usr/bin:$PATH"
 source ~/.profile
 ```
 
-### Run emulator
-Run your app in the emulator using `react-native run-android`.  Now you can
-perform any terminal commands inside the Linux environment, including
-`react-native start`, and the emulator will load the bundle correctly.
+### Possible Issues
+
+#### JAVA_HOME is set to an invalid directory
+If you run into an error that says that 'JAVA_HOME is set to an invalid directory' consider using WSLENV to share the `JAVA_HOME` environment variable with your WSL installation. This only works if you have  Windows 10 Build 17064 or newer. 
```

Comment:
Huh, I can't find much about Build 17064 on the web.

Do you perhaps mean 17063? Here's an announcement of WSLENV:
https://blogs.msdn.microsoft.com/commandline/2017/12/22/share-environment-vars-between-wsl-and-windows/

Reply:
Good catch! I meant Build 17063.